int32 type
uint32 width
uint32 height
uint32 stride
uint32 bytesPP
uint32 p1Offset
uint32 p2Offset
uint32 p3Offset
